contracts/settling_game/library/library_calculator.cairo

```diff
@@ -43,4 +47,58 @@ namespace CALCULATOR:
         end
         return (happiness)
     end
+
+    func calculate_food{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
```

Comment:
No need to implicit args nor alloc_locals in most of the functions in this file.